### PR TITLE
When exporting a set of layers with some duplicate layer names, use a suffix to distinguish

### PR DESCRIPTION
**Pull request by mcilroyc**
Semi-hokey fix for #3609

If you exporting a batch of layers, and there are layer-name collisions, use a simple suffix to avoid a filename collision on the exported assets.

example (updated):
ss

### CODE REVIEW

**Comment by placegraphichere** [comment]
When using "keep both" on OSX, finder uses a space+2 for files. So your example would yield `square 2@3x.png` for the first duplicate, incrementing after that. Perhaps we should follow that pattern instead of `.1`?

**Comment by mcilroyc** [comment]
Updated PR based on the " 2" suggestion from @placegraphichere.
now:
ss

**Comment by chadrolfs** [comment]
I like it.

File: src/js/actions/export.js [hunk 1]
```diff
@@ -170,20 +170,19 @@ define(function (require, exports) {
      * @param {number} assetIndex index of this asset within the layer's list
      * @param {ExportAsset} asset asset to export
      * @param {string=} baseDir Optional directory path in to which assets should be exported
-     * @param {string=} prefix Optional prefix to apply to the fileName
+     * @param {string=} baseFilename Base file name to which things may be prefixed/suffixed
```
Comment (iwehrman) [inline src/js/actions/export.js]: This should be `baseFileName`.

lol j/k.

**Comment by iwehrman** [comment]
Looks good! It might also be nice to have a "warning" symbol in the export dialog if there are layers marked for export with identical names. Adding this number suffix is clearly better than just over-writing all but one of the assets, but ultimately this seems worthy of a warning.

**Pull request merged**